sigopt/__init__.py

```diff
@@ -47,3 +48,12 @@ def set_project(project):
       UserWarning,
     )
   return _global_factory.set_project(project)
+
+def switch_run_context(run_id):
+  if get_run_id() == run_id:
+    return
+
+  context = RunContext(
+    connection = get_connection(),
+    run = get_run(run_id))
+  _global_run_context._run_context = context
```

Comment:
I don't think this needs to be a global function for this PR. Please initialize the run context manually in `tools/generate_feature_importances.py`.

Reply:
Done.